src/styles/colour/default.njk

```diff
@@ -0,0 +1,69 @@
+---
+layout: layout-example.njk
+---
+
+<h2 class="govuk-heading-m">Main colours</h2>
+
+{% for groupName, group in colours.Main %}
+
+  <h3 class="govuk-heading-s">{{groupName}}</h3>
+
+  <table class="govuk-body app-c-colour-list">
+
+    {% for colour in group %}
+
+      <tr class="app-c-colour-list-row">
+
+        <td class="app-c-colour-list-column app-c-colour-list-column--swatch">
+          <span class="app-c-swatch" style="background-color:{{colour.colour}}"></span>
+        </td>
+        <td class="app-c-colour-list-column app-c-colour-list-column--name">
+          {{colour.name}}
+        </td>
+        <td class="app-c-colour-list-column app-c-colour-list-column--colour">
+          {{colour.colour}}
+        </td>
+        <td class="app-c-colour-list-column app-c-colour-list-column--notes">
+          {{colour.notes}}
+        </td>
+
+      </tr>
+
+    {% endfor %}
+
+  </table>
+
+{% endfor %}
+
+<h2 class="govuk-heading-m">Extended colours</h2>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>used for graphs and supporting material</li>
+  <li>for tints of the extended palette use 50% or 25%</li>
+  <li>you can find the departmental colours in the GOV.UK Frontend _colours-organisations file</li>
+</ul>
+
+<table class="govuk-body app-c-colour-list">
```

Comment:
As above, I think if we're going to display this data as a table, we should add column headers.